js/scrollspy.js

```diff
@@ -40,14 +40,15 @@
 
   ScrollSpy.prototype.refresh = function () {
     var that          = this
-    var offsetMethod  = 'offset'
+    var forcePosition = this.options.position
+    var offsetMethod  = forcePosition && 'position' || 'offset'
     var offsetBase    = 0
 
     this.offsets      = []
     this.targets      = []
     this.scrollHeight = this.getScrollHeight()
 
-    if (!$.isWindow(this.$scrollElement[0])) {
+    if (forcePosition === void 0 && !$.isWindow(this.$scrollElement[0])) {
```

Comment:
Historically, we've also compared to undefined with `== null` where the explicit distinction isn't necessary.